src/node_main.cc

```diff
@@ -27,13 +27,25 @@
 #include <VersionHelpers.h>
 #include <WinError.h>
 
+#define SKIP_CHECK_VAR "NODE_SKIP_PLATFORM_CHECK"
+#define SKIP_CHECK_SIZE 1
+#define SKIP_CHECK_VALUE "1"
+
 int wmain(int argc, wchar_t* wargv[]) {
   // Windows Server 2012 (not R2) is supported until 10/10/2023, so we allow it
   // to run in the experimental support tier.
+  char buf[SKIP_CHECK_SIZE + 1];
   if (!IsWindows8Point1OrGreater() &&
-      !(IsWindowsServer() && IsWindows8OrGreater())) {
+      !(IsWindowsServer() && IsWindows8OrGreater()) &&
+      (GetEnvironmentVariableA(SKIP_CHECK_VAR, buf, sizeof(buf)) !=
+       SKIP_CHECK_SIZE ||
+       strncmp(buf, SKIP_CHECK_VALUE, SKIP_CHECK_SIZE + 1) != 0)) {
     fprintf(stderr, "This application is only supported on Windows 8.1, "
```

Comment:
```suggestion
    fprintf(stderr, "Node.js is only supported on Windows 8.1, "
```

Otherwise it brings a confusion to the user: which application? the one that runs on Node.js, Is it coming from a third party module, etc.

(I know this word not touched by this PR, but given that we are changing the behavior here, it makes sense to reduce confusion)